pom.xml

```diff
@@ -1577,6 +1577,13 @@
         <artifactId>jt-shadedrelief</artifactId>
         <version>${jaiext.version}</version>
       </dependency>
+      
+      <!-- Third-party -->
+      <dependency>
```

Comment:
Ah, so mbstyle is using json-simple to parse JSON? The library is completely dead, has not seen releases since 2012.
Not holding it as a blocker for the module, but wondering, is there any plan to switch to something supported?

Reply:
I am not aware why this was chosen, just annoyed it was not in the dependency management section.

Didn't we talk this out with GeoMesa folks to select a decent JSON library?

Reply:
There was a discussion on dependencies for gt-main, and we ended up using jackson for that. But mbstyles is older, and used the same library as the semi-defunct gt-geojson (which I hope is gonna be replaced by the gt-jacksondatastore one day or the other...)